packages/database/lib/migrations/20250325122817_plans.cjs

```diff
@@ -0,0 +1,31 @@
+/**
+ * @param {import('knex').Knex} knex
+ */
+exports.up = async function (knex) {
+    await knex.raw(
+        `CREATE TABLE "plans" (
```

Comment:
I am not a huge fan of the name `plans`. For me plans represents a bundle of features, where here we are storing the features individually. Why not `features`?

Reply:
We are already discussing having stripe in the coming weeks; storing the plan dates will fit nicely in this table. Do you have an alternative suggestion?

Reply:
I haven't part of the discussion regarding stripe. How is this table gonna look like then?

Reply:
Not sure yet but from my experience we just need a few fields
```
stripeCustomerId      
stripeSubscriptionId     
stripePriceId           
stripeCurrentPeriodStart 
stripeCurrentPeriodEnd  
currentPlanId            
```